openmetadata-service/src/main/java/org/openmetadata/service/resources/events/ChangeEventResource.java

```diff
@@ -41,12 +42,12 @@
 import org.openmetadata.service.util.EntityUtil;
 import org.openmetadata.service.util.ResultList;
 
-@Path("/v1/events")
-@Api(value = "Events resource", tags = "events")
+@Path("/v1/changeEvents")
+@Api(value = "Change Events resource", tags = "changeEvents")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
-@Collection(name = "events")
-public class EventResource {
+@Collection(name = "changeEvents")
+public class ChangeEventResource {
```

Comment:
Let's use EventResource. All the APIs provided by this should be called Events API.